WordPress/src/main/java/org/wordpress/android/ui/posts/social/PostSocialSharingModelMapper.kt

```diff
@@ -85,9 +85,9 @@ class PostSocialSharingModelMapper @Inject constructor(
         val sharingEnabledConnections = connections.filter { it.isSharingEnabled }
         return when {
             // No more shares left.
-                shareLimit.sharesRemaining <= 0 -> true
+            shareLimit.sharesRemaining <= 0 -> true
             // Sharing to some accounts, but not enough shares for all.
-            sharingEnabledConnections.isNotEmpty() && shareLimit.sharesRemaining < connections.size -> true
+            connections.size > sharingEnabledConnections.size && shareLimit.sharesRemaining < connections.size -> true
```

Comment:
Just to document it, thanks for explaining on slack that this change is to further align to the spec, and be sure we are only sharing to some (not all) the available connections 🙇